frontend/app/routes/_gcweb-app.update-info.tsx

```diff
@@ -48,33 +49,20 @@ export default function UpdateInfo() {
 
   const fieldErrors = actionData?.errors;
 
+  let navigate = useNavigate();
+
   return (
     <>
       <h1 id="wb-cont" property="name">
         Update personal info
       </h1>
       <p>Please update your info below.</p>
-      <Form method="post">
         <div className="form-group">
-          <label htmlFor="phoneNumber" className="required">
-            <span className="field-name">Phone number</span>
-            <strong className="required mrgn-lft-sm">(required)</strong>
-            {fieldErrors?.phoneNumber?._errors &&
-              fieldErrors.phoneNumber._errors.map((error, idx) => (
-                <span key={idx} className="label label-danger wb-server-error">
-                  <strong>
-                    <span className="prefix">Error:</span>
-                    <span className="mrgn-lft-sm">{error}</span>
-                  </strong>
-                </span>
-              ))}
-          </label>
-          <input id="phoneNumber" name="phoneNumber" className="form-control" maxLength={32} defaultValue={actionData?.formData.phoneNumber ?? loaderData.userInfo.phoneNumber} data-testid="phoneNumber" />
+          <PhoneNumber phoneNumber={actionData?.formData.phoneNumber ?? loaderData.userInfo.phoneNumber} fieldErrors={fieldErrors?.phoneNumber?._errors}/>
         </div>
         <div className="form-group">
-          <button className="btn btn-primary btn-lg">Update info</button>
+          <button className="btn btn-primary btn-lg" onClick={() => navigate("/update-phone-number")}>Edit</button>
```

Comment:
Personally, I prefer this design since it makes it perfectly clear what the action button is:

![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/48123208/ec8c3448-7d8c-4d4d-a788-9384c057b729)

Reply:
FYI this is how WET recommends the order of action buttons:

![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/55502055/b63994ff-b705-4fa3-9090-b3e3f2c25f0e)

See https://wet-boew.github.io/wet-boew-styleguide/design/buttons-en.html

Reply:
My preference:

![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/48123208/43e67a46-b826-43b3-b7c1-ea608c4ddfbc)

Reply:
Will change to link button in next PR